crates/rust/tests/codegen.rs

```diff
@@ -5,6 +5,10 @@ mod codegen_tests {
     macro_rules! codegen_test {
         (wasi_cli $name:tt $test:tt) => {};
         (wasi_http $name:tt $test:tt) => {};
+        (wasi_clocks $name:tt $test:tt) => {};
+        (wasi_filesystem $name:tt $test:tt) => {};
+        (issue569 $name:tt $test:tt) => {};
+        (multiversion $name:tt $test:tt) => {};
```

Comment:
Could you add an option to the macro which says "just `generate` everything" to use in tests here? It's ok to be a top-level option that's a bit of a hack, but it might also be useful for others transitioning from the previous behavior to this new behavior too.